front/src/pages/nft/[id].tsx

```diff
@@ -22,13 +22,25 @@ const SingleNFTPage = (props: SingleNFTPageProps) => {
     }
 
     if (isSuccess) {
-        return <>
-            <GCGrid className={classes.grid}>
-                <GCGridCol sm={12} md={6}>
-                    <GCImage src={data.nftImageUrl} alt="gun" />
-                </GCGridCol>
-            </GCGrid>
-        </>;
+        return (
+            <>
+                <GCGrid className={classes.grid}>
+                    <GCGridCol sm={12} md={6} className={classes.position}>
+                        <div className={classes.nftImage}>
+                            <GCImage src={data.nftImageUrl} alt="gun" />
+                        </div>
+                        <div className={classes.positionText}>
+                            <GCText>{data.name}</GCText>
+                            <div className={classes.listSize}>
+                                <GCList size={12}>
```

Comment:
Please take another look at the layout from Miro. We have a list component. It should contain the following listitem: mintingDate, rarity, collection, firstOwner. All of them are in the data from the request.

Separately, on our visualization there is a price. It wouldn't be bad to take it out separately.

What about the description of the gun , it should be a separate text component under list component.

Reply:
I finished that

Reply:
I agree with Peter, and lets create separate component like nft info for this. @Logovas1994 plz help here

Reply:
Also we should add name of field that is inside info like, 
Minting date 01.01.2023
Rarity USUAL
etc...